### DISCUSSION TITLE
Audio Overlays

### DISCUSSION THREAD

**Discussion by burninc0de** [readium/kotlin-toolkit#116] [Q&A]
Why doesn't the R2-Reader support EPUB3 with media overlays, e.g. TalkingBooks? I tried loading multiple EPUB files with audio that work perfectly fine in Thorium Reader but it doesn't look like the mobile app supports this feature.

**Comment by mickael-menu** [comment]
Media Overlays are not yet supported in the mobile toolkits, but will be, eventually.

**Reply by burninc0de** [reply]
Thanks a lot! Couldn't find anything about it in the description, so I thought I must have mixed something up. Much appreciated.
